distribution/lib/Standard/Database/0.0.0-dev/src/Internal/SQLite/SQLite_Dialect.enso

```diff
@@ -628,6 +626,9 @@ make_custom_cast column target_value_type type_mapping =
     result = Ref.new Nothing
```

Comment:
I find this logic hard to follow. The return value would be either `Nothing` or the return value of `result.put expr` -- but isn't the the return value of `Ref.put` the old value?

Reply:
The return value of `put` is discarded.

The function returns `ref.get` at the very end.

I agree it is not ideal. This is a hack to get multiline if without adding a ton of indentation to the `else` branches. There's just too many branches here (and while sometimes too many branches is bad, here it is just intrinsic to this function) to make it readable.

Once we get multiline `if` I will be more than happy to replace this with just:
```
if cond1 then
    ...
else if cond2 then
    ...
else
    Nothing
```

Reply:
Do you think I should rewrite it with increasing indentation?

I'd rather keep it as is as IMO it is readable once the reasons are explained.